alumnos/leyla/clase02-ejercicios/02.js

```diff
@@ -0,0 +1,10 @@
+function esPar(num) {
+    if (num % 2 === 0 ) {
```

Comment:
En estos casos, puedes devolver directamente la condición 😄  

```suggestion
    return num % 2 === 0
```